### PR TITLE
MET-2033: Magento 2 sku numbers

### PR DESCRIPTION
**Pull request by adamof**
➕ Adds SKU data with `view_product`, `add_to_cart` and `order` events
🔧 Fixes the development installation script

===

Jira story [#MET-2033](https://metrilojira.atlassian.net/browse/MET-2033) in project *Features team*:

### CODE REVIEW

File: Helper/Data.php [hunk 1]
```diff
@@ -197,6 +198,7 @@ public function prepareOrderDetails($order)
                 $mainProduct = $this->productRepository->getById($parentId);
                 $options = (array)$item->getProductOptions();
                 $dataItem['option_id'] = $options['simple_sku'];
+                $dataItem['option_sku'] = $options['simple_sku'];
```
Comment (evgeni-spasov-metrilo) [inline Helper/Data.php]: Soo `$dataItem['option_id']` is the same as `$dataItem['option_sku']`?

**Comment by adamof** [comment]
@evgeni-spasov-metrilo have a look again :)

File: Helper/Data.php (hunk 1 above)
Comment (evgeni-spasov-metrilo) [reply Helper/Data.php]: I'd really like if we have the comment from `addToCart` here also:
`// for legacy reasons - we have been passing the SKU as ID for the child products`

Comment (adamof) [reply Helper/Data.php]: done :)

**Pull request merged**